test/unit/schema.tests.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
-var assume  = require('assume'),
-    joi     = require('joi-of-cql'),
-    helpers = require('../helpers'),
-    schemas = require('../fixtures/schemas'),
-    Schema  = require('../../lib/schema');
+var assume            = require('assume'),
+    datastarTestTools = require('datastar-test-tools'),
+    joi               = require('joi-of-cql'),
+    schemas           = require('../fixtures/schemas'),
+    Schema            = require('../../lib/schema');
 
-var debug  = helpers.debug,
-    cql = joi.cql;
+var helpers = datastarTestTools.helpers;
+
```

Comment:
Usually eslint would complain about this requiring you to split `require` declarations from others.